src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -1872,17 +1856,6 @@ private synchronized void markChangedOrUnChanged() {
         frame.setWindowTitle();
     }
 
-    /**
-     * Selects a single entry, and scrolls the table to center it.
-     *
-     * @param pos Current position of entry to select.
-     */
-    public void selectSingleEntry(int pos) {
```

Comment:
Yes, this method was not used. Probably a similar logic is implemented in the places you mentioned and thus it would be easy to refactor-out a corresponding method. But even then BasePanel is probably not the right place for this method.